ls1intum/Artemis
# `Exam mode`: Fix text exercise score display when two correction rounds are enabled

### Checklist
#### General
#### Server

### Motivation and Context
Fixes #7124

### Description
The text editor endpoint was sending all results to the client, however, the client only expected to receive one. This lead to an inconsistent amout of points visible to the student

### Steps for Testing
Prerequisites:
- 1 Student
- 1 Exam with 1 text exercise, second correction enabled (two different scores in first and second correction round)

1. View the exam summary and press "view result".
2. Check that the points view shows the correct values (same as the result view)

### Review Progress

#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage

### Screenshots
see issue
